instrumentation/rocketmq-client/src/main/java/brave/rocketmq/client/RocketMQTracing.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package brave.rocketmq.client;
+
+import brave.Span;
+import brave.Tracer;
+import brave.Tracing;
+import brave.messaging.MessagingRequest;
+import brave.messaging.MessagingTracing;
+import brave.propagation.Propagation;
+import brave.propagation.TraceContext.Extractor;
+import brave.propagation.TraceContext.Injector;
+import brave.propagation.TraceContextOrSamplingFlags;
+import brave.sampler.SamplerFunction;
+import org.apache.rocketmq.client.consumer.listener.MessageListenerConcurrently;
+import org.apache.rocketmq.client.consumer.listener.MessageListenerOrderly;
+import org.apache.rocketmq.client.hook.SendMessageHook;
+
+import java.util.Map;
+
+/** Use this class to decorate your RocketMQ consumer / producer and enable Tracing. */
+public final class RocketMQTracing {
+  static final String
+    ROCKETMQ_TOPIC = "rocketmq.topic",
+    ROCKETMQ_TAGS = "rocketmq.tags";
+
+  public static RocketMQTracing create(Tracing tracing) {
+    return newBuilder(tracing).build();
+  }
+
+  public static Builder newBuilder(Tracing tracing) {
+    return newBuilder(MessagingTracing.create(tracing));
+  }
+
+  /** @since 5.9 */
```

Comment:
this will be 6.1 here and everywhere. In fact often we update the poms (including bom) to the next snapshot when adding a new feature also